src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -344,6 +349,20 @@ class GoogleDataprocDAO(protected val dataprocConfig: DataprocConfig,
     executeGoogleRequestAsync(googleProject, "Bucket Path " + bucketPath.toUri, fileInserter).void
   }
 
+  /* set the pet service account as the staging bucket owner */
+  override def setStagingBucketOwnership(cluster: LeoCluster): Future[Unit] = {
+    val acl = new BucketAccessControl().setEntity(cluster.googleServiceAccount.value).setRole("OWNER")
```

Comment:
Note there is `BucketAccessControl` (which gives access to list the bucket) and `ObjectAccessControl` which gives access to objects inside the bucket. Probably want both in this case. See `GoogleDataprocDAO.createBucket` for an example.

Reply:
Ah - I misunderstood how that works.  Thanks.

Reply:
Also the entity string should be `s"user-${cluster.googleServiceAccount.value}"`.